Specifications/Events/for_EventConverter/when_converting_a_committed_event_to_sdk/from_a_commit_events_response.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.Protobuf.Contracts;
+using Dolittle.Runtime.Events.Contracts;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Dolittle.SDK.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Machine.Specifications;
+using Newtonsoft.Json;
+using PbArtifact = Dolittle.Artifacts.Contracts.Artifact;
+using PbCommittedEvent = Dolittle.Runtime.Events.Contracts.CommittedEvent;
+
+namespace Dolittle.SDK.Events.for_EventConverter.when_converting_a_committed_event_to_sdk
+{
+    public class from_a_commit_events_response : given.a_converter_and_a_protobuf_execution_context
```

Comment:
Rename the test to:
```suggestion
namespace Dolittle.SDK.Events.for_EventConverter
{
    public class when_converting_a_commit_events_response_to_sdk : given.a_converter_and_a_protobuf_execution_context
```
Also move the file to match the name